src/main/java/duke/Duke.java

```diff
@@ -48,58 +48,55 @@ protected String getResponse(String input) {
             responseMessage = taskList.taskMessage;
         } else if (command.equals("done")) {
```

Comment:
What about using enum class instead of direct setting up the text commands here?